contracts/near/eth-client/build.sh

```diff
@@ -12,6 +12,7 @@ else
      userflag=""
 fi
 
+# simple:latest \
```

Comment:
You can remove this change to keep diff smaller

Reply:
ok